assesspy/tests/test_metrics.py

```diff
@@ -0,0 +1,48 @@
+import pytest as pt
+
+import assesspy as ap
+
+
+class TestMetrics:
+    @pt.fixture(params=["cod", "prd", "prb", "mki", "ki"])
+    def metric(self, request):
+        return request.param
+
+    @pt.fixture
+    def metric_val(self, metric, ccao_data, quintos_data):
+        if metric in ["mki", "ki"]:
+            return getattr(ap, metric)(*quintos_data)
+        return getattr(ap, metric)(*ccao_data)
+
+    def test_metric_value_is_correct(self, metric, metric_val):
+        expected = {
+            "cod": 17.81456901196891,
+            "prd": 1.0484192615223522,
+            "prb": 0.0009470721642262903,
+            "mki": 0.794,
+            "ki": -0.06,
+        }
+        assert pt.approx(metric_val, rel=0.02) == expected[metric]
```

Comment:
IMO it's worth updating these tests to use known good values rather than just values resulting from our sample data. I opened #25 for this purpose.